### PR TITLE
generating a link if parent is a http url

### LINKED ISSUE #1419: Link to repo upstream web pages
It would be great if SCM upstreams with a `https://` address as depicted above were turned into hyperlinks on the front page.

### PR DESCRIPTION
**Pull request by tulinkry**
fixes #1419

### CODE REVIEW

File: web/repos.jspf [hunk 1]
```diff
@@ -185,7 +186,12 @@ Copyright (c) 2015, 2017, Oracle and/or its affiliates. All rights reserved.
                             if (currentVersion == null) {
                                 currentVersion = "N/A";
                             }
-                            %><td><%= Util.htmlize(type) %>: <%= Util.htmlize(parent) %> (<%= Util.htmlize(branch) %>)</td><%
+                                %><td><%= Util.htmlize(type) %>: <%
+                                if(Util.isHttpUrl(parent)) {
```
Comment (vladak) [inline web/repos.jspf]: missing whitespace after `if` ?

File: web/repos.jspf [hunk 2]
```diff
@@ -312,7 +318,12 @@ Copyright (c) 2015, 2017, Oracle and/or its affiliates. All rights reserved.
                                 if (currentVersion == null) {
                                     currentVersion = "N/A";
                                 }
-                                %><td><%= Util.htmlize(type) %>: <%= Util.htmlize(parent) %> (<%= Util.htmlize(branch) %>)</td><%
+                                %><td><%= Util.htmlize(type) %>: <%
+                                if(Util.isHttpUrl(parent)) {
```
Comment (vladak) [inline web/repos.jspf]: ditto

**Comment by tulinkry** [comment]
I did a util method to generate the whole link

File: web/repos.jspf (hunk 2 above)
Comment (tulinkry) [inline web/repos.jspf]: I changed it completely

File: web/repos.jspf (hunk 1 above)
Comment (tulinkry) [inline web/repos.jspf]: I changed it completely

**Pull request merged**